tests/unit/introspection.tcl

```diff
@@ -449,6 +449,30 @@ start_server {tags {"introspection"}} {
         assert {[dict exists $res bind]}  
     }
 
+    test {redis-server command line arguments - NON_MULTI_ARG only one arg} {
+        assert_error {*port*wrong number of arguments*} {exec src/redis-server --port}
+        assert_error {*port*wrong number of arguments*} {exec src/redis-server --port 6379 6380}
+        assert_error {*port*wrong number of arguments*} {exec src/redis-server --port --enable-debug-command yes}
+        assert_error {*port*wrong number of arguments*} {exec src/redis-server --port --bla}
+    } {} {external:skip}
+
+    test {redis-server command line arguments - MULTI_ARG consume at least one arg} {
```

Comment:
let's test the happy path too.
i.e. of a multi-arg with and without quotes.
we'll need to either allocate a port to connect to and validate the config.
or just rely on parsing stdout and using signal to terminate it.

Reply:
Yes I want to add it before, but don't know how to verify it... so i skip it and publish the PR first, then seek the suggestion

Reply:
I added some tests